atoma-state/src/state_manager.rs

```diff
@@ -1509,23 +1518,24 @@ impl AtomaState {
         stack_small_id: i64,
         estimated_total_compute_units: i64,
         total_compute_units: i64,
-    ) -> Result<()> {
+    ) -> Result<(f64, i64)> {
```

Comment:
change the rustdoc with description of what is returned

Reply:
I honestly think we should get rid of all the tuples in the code, they are harder to read compared to struct

Reply:
Hum, will crate a struct then, thanks for the suggestion